Doc/library/argparse.rst

```diff
@@ -192,6 +192,11 @@ arguments it contains. The default message can be overridden with the
 The ``%(prog)s`` format specifier is available to fill in the program name in
 your usage messages.
 
+When a custom usage message is specified for the main parser, consider
+also passing the ``prog`` argument to :meth:`~ArgumentParser.add_subparsers`
+or the ``prog`` and the ``usage`` arguments to
+:meth:`~_SubParsersAction.add_parser`.
```

Comment:
Isn't it obvious? For the same reason they use `usage` in the main parser. Do you mind to provide exact text you want to add?

I can also remove this paragraph if it is not helpful.

Reply:
It may be to some, but I think explaining the UX implications more clearly here is also valuable to newer users. I do think the blurb is a good addition all up though.